WatsonWebserver/Parameters/HttpGetAttribute.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace WatsonWebserver
+{
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = false)]
+    public sealed class HttpGetAttribute : Attribute
+    {
+        public string MethodName;
```

Comment:
Is null a valid value for this?  If not I will wrap it with a String.IsNullOrEmpty check